### PR TITLE
Add Bootstrap v5 support for accordions

### PR DESCRIPTION
**Pull request by simonihmig**
Had to refactor the tests (selectors) a bit, as they were not *quite* right, and failed for BS5 (e.g. assumed you were clicking on the heading, instead of the button contained within it).

Didn't touch the a11y markup (yet), needs to be fixed separately, see #999.

### CODE REVIEW

File: addon/components/bs-accordion/item/title.hbs [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+{{else}}
+  <div
+    class="{{if (macroCondition (macroGetOwnConfig "isBS3")) "panel-heading"}} {{if (macroCondition (macroGetOwnConfig "isBS4")) "card-header"}}"
+    role="tab"
+    ...attributes
+    {{on "click" this.handleClick}}
+  >
+    {{#if (macroCondition (macroGetOwnConfig "isBS4"))}}
+      <h5 class="mb-0">
+        <button class="btn btn-link {{if @disabled "disabled"}} {{if @collapsed "collapsed"}}" type="button" disabled={{@disabled}}>
```
Comment (simonihmig) [inline addon/components/bs-accordion/item/title.hbs:27]: Had to change BS3/4 markup a little, putting `.collapsed` on the button/a instead of the parent div. This is actually expected (see e.g. https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example), so could be seen here as a bug fix. Though it might theoretically cause some regressions for users... 🤔

It does not affect the CSS styling in any way for BS3/4 AFAICT, but it does for BS5, so had to be changed.

File: addon/components/bs-accordion/item/title.hbs [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+          {{yield}}
+        </button>
+      </h5>
+    {{/if}}
+    {{#if (macroCondition (macroGetOwnConfig "isBS3"))}}
+      <h4 class="panel-title">
+        <a href="#" class="{{if @disabled "disabled"}} {{if @collapsed "collapsed"}}" disabled={{@disabled}}>
```
Comment (jelhan) [inline addon/components/bs-accordion/item/title.hbs:34]: This is also fixing a bug independently from Bootstrap 5 support, isn't it?

**Review by jelhan — COMMENTED** [review]
I'm very sorry that review took that long. To be honest I fully missed this one.

File: addon/components/bs-accordion/item/title.hbs (hunk 1 above)
Comment (simonihmig) [reply addon/components/bs-accordion/item/title.hbs:27]: > So I guess we don't need to address that bug here.

You are right. Extracted the changes to #1590.

Will rebase this PR once the fix has landed.

Comment (simonihmig) [reply addon/components/bs-accordion/item/title.hbs]: @jelhan done now! The diff is now much smaller, and confined to only BS5 related changes!

File: addon/components/bs-accordion/item/title.hbs [hunk 3]
```diff
@@ -1,23 +1,40 @@
...
+    {{#if (macroCondition (macroGetOwnConfig "isBS4"))}}
+      <h5 class="mb-0">
+        <button class="btn btn-link {{if @disabled "disabled"}} {{if @collapsed "collapsed" "expanded"}}" type="button" disabled={{@disabled}}>
+          {{yield}}
+        </button>
+      </h5>
+    {{/if}}
+    {{#if (macroCondition (macroGetOwnConfig "isBS3"))}}
+      <h4 class="panel-title">
+        <a href="#" class="{{if @disabled "disabled"}} {{if @collapsed "collapsed" "expanded"}}" disabled={{@disabled}}>
```
Comment (jelhan) [inline addon/components/bs-accordion/item/title.hbs:34]: Did you added the `disabled` attribute intentionally? I don't think it is suported for link elements (`<a>`). Also it does not seem to be related to Bootstrap 5 support.

**Review by jelhan — CHANGES_REQUESTED** [review]
Much easier to review after the bug fixes has been extracted into separate merge requests. Thanks a lot!

Just a small question left. Beside that everything looks good.

File: addon/components/bs-accordion/item/title.hbs (hunk 3 above)
Comment (simonihmig) [reply addon/components/bs-accordion/item/title.hbs]: Hm, was probably left over from the previous monolithic PR. Removed it now, good catch!

**Review by jelhan — APPROVED** [review]
